File: BleModule/src/main/java/com/topdon/lms/sdk/utils/StringUtils.java
package com.topdon.lms.sdk.utils;

/**
 * Stub implementation for StringUtils
 * 
 * @author Copilot
 */
public class StringUtils {
    public static boolean isEmpty(String str) {
        return str == null || str.trim().isEmpty();
    }
    
    public static boolean isNotEmpty(String str) {
        return !isEmpty(str);
    }
}

File: android/libapp/src/main/java/com/topdon/lms/utils/StringUtils.kt
package com.topdon.lms.utils

/**
 * Stub implementation of StringUtils for compilation
 */
object StringUtils {
    @JvmStatic
    fun isEmpty(str: String?): Boolean = str.isNullOrEmpty()
    
    @JvmStatic
    fun isNotEmpty(str: String?): Boolean = !str.isNullOrEmpty()
    
    @JvmStatic
    fun equals(str1: String?, str2: String?): Boolean = str1 == str2
    
    @JvmStatic
    fun format(format: String, vararg args: Any): String = String.format(format, *args)
}